### ISSUE TITLE
Can't close opened doors

### ISSUE THREAD

**Issue by rlgenesis** [apollo-rsps/apollo#317]
Basically any door that is already opened can't be closed on Apollo. On RuneScape you can close the majority of these doors with a few exceptions (and with those exceptions you get a message saying "This door is stuck" or something along those lines)

http://pix.toile-libre.org/upload/original/1493155642.png

**Comment by WizardJesse1** [comment]
They can be closed the id of the object is probably just not supported which is why its not closing

in plugin/navigation/doors/constants

you will see

`  # Door object ids that have a hinge on the left side.
  LEFT_HINGE_DOORS = Set.new [1516, 1536, 1533]

  # Door object ids that have a hinge on the right side.
  RIGHT_HINGE_DOORS = Set.new [1519, 1530, 4465, 4467, 3014, 3017, 3018, 3019]`

if you printed out the id of the opened door its probably not here.

**Comment by Major-** [comment]
Jesse is correct (id for the door in question is `1531`), although the current door code doesn't work with doors that are open by default. Looking into this.

We also don't support double doors, but thats a separate problem.

**Comment by tlf30** [comment]
I think this is still an issue in the kotlin version that @arinhouck #356 contributed.

**Comment by Major-** [comment]
Yes, it is, the doors plugin doesn't work properly and doesn't support similar mechanisms like gates and curtains. I know how to implement it properly but it's not going to happen until we merge `kotlin-experiments` into `master` - the current plugin will do for now
